earth2studio/data/rx.py

```diff
@@ -234,9 +234,11 @@ def __call__(
         return super().__call__(time, variable)
 
 
-class CosineSolarZenith:
+class CosineSolarZenith:  # pragma: no cover
```

Comment:
Adding here because this needs physicsnemo import and I dont want to add physicsnemo into data optional group